app/constants/DR/baseUrls.js

```diff
@@ -1,8 +1,6 @@
 import env from 'react-native-config';
 
-export const MEPYD_C5I_SERVICE = __DEV__
-  ? 'https://webapps.mepyd.gob.do'
-  : env.MEPYD_C5I_URL.replace('" #ignoreline', '');
+export const MEPYD_C5I_SERVICE = env.MEPYD_C5I_URL.replace('" #ignoreline', '');
```

Comment:
I don't think this should be merge like this, remember this URL is for production only, there's should be a development mode, please put it the way it was

Reply:
I tested that the real difference is in .env and .env.release, not in the baseURL

Reply:
ya, fixed